### PR TITLE
Adding mobile menu to portfolio project

### PR DESCRIPTION
**Pull request by imranhossainemi**
Updated linter file form html-css to html-css-js. 
Add main.js file 
Updating imgae , index.html and main.css files
Added Mobile menu functionality using java script

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -34,13 +35,57 @@ body {
...
+  transition: 0.3s;
+}
+
+.header .navSec nav .navigation .nav-item {
+  width: 85%;
+  height: 50px;
+  list-style: none;
+  margin-left: 50px;
+}
+
```
Comment (Owusu-Desmond) [inline main.css]: - [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - Make sure to make the welcome text invisible when the mobile menu appears to match the Figma design.
  - Make the close button size and design match the Figma design.
  - Make sure the size of the navigation links matches the Figma design.

| Your Design |Figma Design |
| :---:   | :---: | 
| Screenshot from 2023-01-31 08-45-52| Screenshot from 2023-01-31 08-36-17|

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @imranhossainemi and @dheerajsachdeva , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great work adding the js linters ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by prolajumokeoni** [comment]
Hi @Team 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted the wrong pr, for Portfolio: details popup window milestone

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Olamarx — APPROVED** [review]
Hi @imranhossainemi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No Linters error ✅
- Descriptive pull request✅
- Good commit messages ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**